src/HospitalAPI/Controllers/Public/AuthController.cs

```diff
@@ -23,7 +23,7 @@ public async Task<IActionResult> RegisterPatient(CreatePatientDTO createPatientD
         {
             User user = createPatientDTO.MapUserToModel();
             user.Person = createPatientDTO.MapPatientToModel();
-            User createdUser = await _authService.RegisterPatient(user,createPatientDTO.Allergens);
+            User createdUser = await _authService.RegisterPatient(user,createPatientDTO.Allergens,createPatientDTO.DoctorUid);
```

Comment:
Instead of sending separate things, you can send user and createPatientDTO. This way method gets more parameters without need for them.